buildSrc/src/main/java/Config.kt

```diff
@@ -49,11 +49,9 @@ object Config {
     object Sentry {
         val SENTRY_CLIENT_NAME = "sentry.java.android"
         val group = "io.sentry"
-//        TODO: change version to publish new version
-        val version = "2.0.0-rc03"
+        val version = "2.0.0-rc03" // updated by craft
         val description = "SDK for sentry.io"
-//        TODO: change version code to publish new version, follow the pattern of `version`
-        val buildVersionCode = 20014
+        val buildVersionCode = 20014 // updated by craft
```

Comment:
I'd recommend moving these properties away from here to https://github.com/getsentry/sentry-android/blob/master/gradle.properties

Reply:
What is the benefit?

Reply:
not much tbh, it's a key/value props file and it'd be easier to change with a CLI tool